src/App/Hub/Endpoints/Egress/hub-egress.test.js

```diff
@@ -107,133 +108,6 @@ describe('Test the dfsp egress endpoints actions', () => {
   });
 });
 
-describe('Test the dfsp egress endpoints thunk actions', () => {
```

Comment:
It seems lot of tests are removed. How did we achieve code coverage after removing all these tests?